db/migrate/20240108211241_fix_orchestration_stack_configuration_script_reference.rb

```diff
@@ -0,0 +1,22 @@
+class FixOrchestrationStackConfigurationScriptReference < ActiveRecord::Migration[6.1]
+  class OrchestrationStack < ActiveRecord::Base
+    include ActiveRecord::IdRegions
+    self.inheritance_column = :_type_disabled
+  end
+
+  TYPES = %w[ManageIQ::Providers::Awx::AutomationManager::Job ManageIQ::Providers::AnsibleTower::AutomationManager::Job].freeze
+
+  def up
+    say_with_time("Fixing OrchestrationStack#configuration_script foreign key") do
+      OrchestrationStack.in_my_region.where(:type => TYPES).update_all('configuration_script_id = orchestration_template_id')
+      OrchestrationStack.in_my_region.where(:type => TYPES).update_all(:orchestration_template_id => nil)
```

Comment:
I think think can be done in a single line:

```suggestion
      OrchestrationStack.in_my_region.where(:type => TYPES).update_all(
        :configuration_script_id   => :orchestration_template_id,
        :orchestration_template_id => nil
      )
```

Reply:
Pretty sure I tried that and it didn't work...let me double check

Reply:
Yeah no, to set one column to the value of another you need the `'columnA = columnB'` syntax, `update_all(:configuration_script_id   => :orchestration_template_id)` just sets `:configuration_script_id=0` in this case

Reply:
Plus I figured they're in the same transaction so the performance difference will be negligible

Reply:
oh wild, I was sure that worked 🤯

Reply:
Yeah I'm assuming AR can't tell the difference between you wanting to set it to the column `:orchestration_template_id` or the symbol value `:orchestration_template_id`